core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -122,25 +121,21 @@ object EntityDecoder extends EntityDecoderInstances {
   /** Create a new [[EntityDecoder]]
     *
     * The new [[EntityDecoder]] will attempt to decode messages of type `T`
-    * only if the [[Message]] satisfies the provided [[MediaRange]]s
+    * only if the [[Message]] satisfies the provided [[MediaRange]]
+    * note: Sync bound to note that such an `F[_]` may throw
```

Comment:
Isn't the bound still `Applicative` or is it referring to something else?

Reply:
Good catch.  I took a stab at correcting it.